Raspi/Test/MockSensor.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+
+class MockSensor:
+    """ Base class to make a mock sensor that produces random data"""
+
+    def __init__(self, *ave_val, percent_range=0.05, decimals=2):
```

Comment:
I think I would prefer to rename the variable `ave_val` to `average_value`. Just because, at a glance, I did not know what `ave_val` meant. I had to scroll up to this documentation to understand what it represented too. You spend more time reading code than you do writing code, so make it easy to read!

Reply:
I agree, it would be confusing if you don't know what it stands for.